trnsysGUI/mainWindow.py

```diff
@@ -466,13 +472,30 @@ def openFile(self):
         maybeCancelled = _prj.getLoadOrMigrateProject()
         if _ccl.isCancelled(maybeCancelled):
             return
+        UserSettings.setRecentProjectJsonPath(self.editor.diagramPath)
         project = _ccl.value(maybeCancelled)
 
         self._resetEditor(project)
 
         if isinstance(project, _prj.MigrateProject):
             self.editor.save()
 
+    def openRecentFile(self):
```

Comment:
This seems like duplication of the usual functionality when opening another project.
Can this be integrated?

Reply:
not exactly, we can defintely extract message box